Ladybird/AppKit/UI/TabController.mm

```diff
@@ -313,10 +313,14 @@ - (void)setUserAgentSpoof:(NSMenuItem*)sender
 
 - (NSButton*)create_button:(NSImageName)image
                with_action:(nonnull SEL)action
+              with_tooltip:(NSString*) tooltip
```

Comment:
spacing looks off here compared to the calls below. We should probably update the `lint-clang-format` check to make sure to check .mm files. Unless this is what clang-format says is correct for the function arguments? (aligned to : )

Reply:
Looking at the rest of the code it seems we align at colon. Changed the calls to do the same.